src/odm_sharing/sharing.py

```diff
@@ -62,7 +63,30 @@ def get_data(c: Connection, tq: TableQuery) -> pandas.DataFrame:
     :raises DataSourceError: if an error occured while retrieving data
     '''
     dq = tq.data_query
-    return cons.exec(c, dq.sql, dq.args)
+    df = cons.exec(c, dq.sql, dq.args)
+
+    # At this point bool values are 0/1, so we have to convert it to
+    # FALSE/TRUE (which is the ODM standard).
+    F = 'FALSE'
+    T = 'TRUE'
+    for col in c.bool_cols[tq.table_name]:
+        kind = df[col].dtype
+        if kind == object:  # str
+            df[col] = df[col].replace({'0': F, '1': T})
+        elif kind == np.int64:
+            df[col] = df[col].astype(str).replace({'0': F, '1': T})
+        elif kind == np.float64:
+            df[col] = df[col].astype(str).replace(
+                {'nan': '', '0.0': F, '1.0': T})
+        else:
+            assert False, f'invalid bool type {kind}'
+
+    # convert None to empty string if the column is string
```

Comment:
Bool columns may end up storing empty values as NULL in the database, which in turn are extracted as None. It doesn't make sense to return `None` values to the user since that's not relevant to the data the user is working with. The input is text and the output is text, where the closest to `None` would be an empty string, which is what I'm normalizing it to here. I've added a comment in the code to explain it better.

Reply:
I see now that any empty cell from excel may be loaded as a None value, so this may happen to non-bool cols as well. I'll add an issue to investigate output values from the API so that the user doesn't have to wrangle with both None and empty string.

Reply:
#42

Reply:
I think you're talking about something else. I meant that in the sharing schema the user can say they want to only keep columns with empty strings. If pandas converts those to `None` then won't that rule not work? Unless you convert empty strings within the schema to None also?

Reply:
not sure, I made an issue for testing that (#43)